Cargo.toml

```diff
@@ -49,10 +49,10 @@ openssl = "0.10.51"
 prost = "0.12.1"
 prost-types = "0.12.1"
 protobuf-src = "1.1.0"
-quinn = "0.9"
-rand = "0.8.5"
+quinn = "0.10"
+rand = "0.7"
```

Comment:
why does this need downgrading?

Reply:
i think the rand change can be reverted unless solana libs need it

Reply:
https://github.com/search?q=repo%3Aquinn-rs%2Fquinn+%22rand+%3D%22&type=code